### PR TITLE
Support Link Promo type in 'standard' format

### LINKED ISSUE #2990: Support Link Promo type in 'standard' format.
A link promo is a different type of promo that can have various formats. At first we only want to support the 'standard' format, and it should use the existing Story Promo, as it is. 

See details in WSFRONTPGS-182

**Describe the solution you'd like**
These promos should be displayed using the existing story promo component.
Link Promos have cps type of `PRO`, however the data we receive has a different format than the normal STY.

```
{
  "name": "Ina mọnamọna: Ijọba rawọ bẹ araalu fun ina ọba to lọ lọjọ isẹgun",
  "summary": "Idakureku ina ọba kii se ajeji s'awọn ọmọ orilẹ-ede Naijiria, sugbọn nigba ti gbogbo ina orilẹ-ede yii ba dede ku lọwọkannaa, ko le sai fa awuyewuye laarin awọ ọmọ orilẹ-ede Naijiria",
  "indexImage": {
    "id": "63640284",
    "subType": "index",
    "href": "http://b.files.bbci.co.uk/BC4C/test/_63640284_babatunde-rajifasholaministafunrinamnamna.jpg",
    "path": "/cpsdevpb/BC4C/test/_63640284_babatunde-rajifasholaministafunrinamnamna.jpg",
    "height": 549,
    "width": 976,
    "altText": "Babatunde Fashola",
    "caption": "Babatunde Fashola",
    "copyrightHolder": "Babatunde Fashola/Facebook"
  },
  "uri": "https://www.test.bbc.com/yoruba/afrika-23176202",
  "contentType": "Text",
  "assetTypeCode": "PRO",
  "timestamp": 1561381319000,
  "type": "link"
}
```

The story promo container will have to be updated so it can use information provided by CPS. There will be some mapping required between `headline` => `name`, etc.

[The CPS whitelist filter](https://github.com/bbc/simorgh/blob/spike-event-tracking/src/app/lib/utilities/preprocessor/rules/cpstypes.js) will also have to be updated since Link Promos use `assetTypeCode` instead of the usual `cpsType`.

**Describe alternatives you've considered**
-

**Testing notes**
- Cypress tests will be needed
- Ensure fixture data has the data necessary
- Exploratory and Compatibility testing required

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### PR DESCRIPTION
**Pull request by Bopchy**
Resolves #2990 

**Overall change:** _Filter both unknown cpsTypes and assetTypeCodes. Add `standard link promo` story._

**Code changes:**

- _Rename `cpstypes.js` to `filterContentType.js`_
- _Filter both unknown cpsTypes and assetTypeCodes in filterUnknownContentTypes()._
- _Add tests for filterUnknownContentTypes()._
- _Add `standard link promo` story._
---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: src/app/containers/StoryPromo/LinkContents/index.jsx [hunk 1]
```diff
@@ -16,8 +16,13 @@ const LinkContents = ({ item }) => {
...
   const headlines = pathOr(null, ['headlines'], item);
-  const { headline, overtyped } = headlines;
-  const content = overtyped || headline;
+  let content;
+  if (headlines === null) {
+    content = pathOr(null, ['name'], item);
+  } else {
+    const { headline, overtyped } = headlines;
+    content = overtyped || headline;
+  }
```
Comment (DenisHdz) [inline src/app/containers/StoryPromo/LinkContents/index.jsx:29]:
```
const getContent = () => {
    if (headlines === null) {
      return pathOr(null, ['name'], item);
    }
    const { headline, overtyped } = headlines;
    return overtyped || headline;
  };

  const content = getContent();
```

**Review by DenisHdz — CHANGES_REQUESTED** [review]
LGTM, just one refactor :)

**Review by j-pendlebury — APPROVED** [review]
Woo looks great 😄👍

**Review by DenisHdz — APPROVED** [review]

**Comment by david-boydell** [comment]
Looks good, this is ready for merge.

**Pull request merged**